### DISCUSSION TITLE
How to disable subscription possibility?

### DISCUSSION THREAD

**Discussion by hexwit** [graphql-java/graphql-java#2411] [Q&A]
Hi.

During instantiating GraphQL object I must provide subscription strategy. 
But what if I don't need subscriptions? 

Is it possible to disable them? Subscription strategy cannot be null as I see.

Regards.

**Comment by bbakerman** [comment]
Just dont put in a subscription type in your schema

```
type Query {
   f :String
}

type Mutation {
  m : String
}

```
